thumbnail-image/index.md

```diff
@@ -0,0 +1,17 @@
+## What images can I use for thumbnails?
+
+Here are the technical requirements and recommendations for thumbnail images:
+
+- The minimum dimensions are 640x360 (recommended minimum of 1280x720 to look decent on 4K displays). 
+- The maximum width and height is 4096x4096. 
+- Your original image should be as large as possible (up to 4096x4096). 
+- The aspect ratio must be 16x9. If it's not 16x9, you can use the handy cropping tool that we've provided. Alternatively, you can crop it with your tool of choice before uploading. 
+    - _Tip:_ This will help if you're not using our cropping tool. You can find the necessary height by taking the image's current width and multiplying it by 0.5625. Round up. 
+- The maximum file size is 3MB. If your file is larger, try compressing and/or optimizing the image. 
+- Your file must be a PNG, JPG, GIF, or animated GIF. PNGs generally look the best. 
+
+[What is a thumbnail image?](Link)
+
+[Choosing a thumbnail image](Link)
```

Comment:
might need to update the above 2 links once those two are merged into 1 question.